src/main/java/com/theelixrlabs/healthcare/service/DoctorService.java

```diff
@@ -52,21 +53,22 @@ public DoctorDto saveDoctor(DoctorDto doctorDto) throws CustomException {
         return savedDoctorDto;
     }
 
-    /**
-     * Custom validation method for checking regex pattern of firstname lastname and Aadhaar number
-     *
-     * @param doctorDto : DoctorDto object containing doctor information.
-     * @throws CustomException : Class to handle custom exception
-     */
     private void validateDoctor(DoctorDto doctorDto) throws CustomException {
-        if (!(doctorDto.getFirstName().matches(DoctorConstants.CHARACTER_ONLY_REGEX_PATTERN))) {
-            throw new CustomException(DoctorConstants.FIRST_NAME_MUST_BE_CHARACTER);
+
+        if (doctorDto.getFirstName().isEmpty()) {
```

Comment:
since you are using @NotBlank field annotation so it can handle the empty check as well 
Please test and apply change for others

Reply:
if I use @valid then it will return as all fields as mandatory. Since endpoint is patch i am using doctorValidation method to validate all the fields separately.